include/aikido/robot/ConcreteRobot.hpp

```diff
@@ -199,7 +207,8 @@ class ConcreteRobot : public Robot
       const aikido::constraint::dart::TSRPtr& tsr,
       const aikido::constraint::dart::CollisionFreePtr& collisionFree,
       double timelimit,
-      std::size_t maxNumTrials);
+      std::size_t maxNumTrials,
+      const Eigen::VectorXd& nominalPosition = Eigen::VectorXd(0));
```

Comment:
Are we keeping this argument? I propose to not, and if we are, we need to create an issue to track it and remove it, and add docstring.